.ddev/config.yaml

```diff
@@ -1,7 +1,7 @@
 name: reference-typoscript
 type: php
 docroot: ""
-php_version: "8.1"
+php_version: "8.2"
```

Comment:
This change is due to v13/main also requiring PHP 8.2+, right?

Reply:
yeah, as-was the sources were not installable